plugin/modify-env/modify-env.py

```diff
@@ -0,0 +1,47 @@
+# Copyright (C) 2015 Kyle Harrigan (kwharrigan@gmail.com)
```

Comment:
@kwharrigan:  For DMTCP version 3.0, we're all planning to change the copyright license from the current GNU LGPL to Apache, version 2.  Would you mind changing the copyright notice above to the following template?  (Still in your name, your e-mail, of course.)  This will save us having to go back and check this at a later date.  Thanks.
```
Copyright [yyyy] [name of copyright owner]

Licensed under the Apache License, Version 2.0 (the "License");
you may not use this file except in compliance with the License.
You may obtain a copy of the License at

    http://www.apache.org/licenses/LICENSE-2.0

Unless required by applicable law or agreed to in writing, software
distributed under the License is distributed on an "AS IS" BASIS,
WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
See the License for the specific language governing permissions and
limitations under the License.
```